content/en/docs/languages/python/propagation.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Propagation
+description: Context propagation for the Python SDK
+aliases: [api/propagation]
+weight: 65
+cSpell:ignore: tracestate
+---
+
+Propagation is the mechanism that moves data between services and processes.
+Although not limited to tracing, it is what allows traces to build causal
+information about a system across services that are arbitrarily distributed
+across process and network boundaries.
+
+## Manual W3C Trace Context Propagation
+
+In some cases, it is not possible to propagate context with an instrumentation
+library. There may not be an instrumentation library that matches a library
+you're using to have services communicate with one another. Or you many have
+requirements that instrumentation libraries cannot fulfill, even if they exist.
+
+When you must propagate context manually, you can use the
+[context API](/docs/instrumentation/js/context).
+
+The following generic example demonstrates how you can propagate trace context
+manually.
+
+First, on the sending service, you'll need to inject the current `context`:
+
+```python
+from flask import Flask
+import requests
+from opentelemetry import trace, propagators, baggage
+from opentelemetry.instrumentation.flask import FlaskInstrumentor
+from opentelemetry.instrumentation.requests import RequestsInstrumentor
+from opentelemetry.trace.propagation.tracecontext import TraceContextTextMapPropagator
+from opentelemetry.baggage.propagation import W3CBaggagePropagator
+from opentelemetry.sdk.trace import TracerProvider
+from opentelemetry.sdk.trace.export import ConsoleSpanExporter, BatchSpanProcessor
+from opentelemetry.exporter.jaeger.thrift import JaegerExporter
+
+app = Flask(__name__)
+
+trace.set_tracer_provider(TracerProvider())
+trace.get_tracer_provider().add_span_processor(BatchSpanProcessor(ConsoleSpanExporter()))
+
+jaeger_exporter = JaegerExporter(
+    agent_host_name="localhost",  # Set your Jaeger agent host
+    agent_port=6831,  # Set your Jaeger agent port
+)
```

Comment:
JaegerExporter is deprecated, please use OTLP

Reply:
Done.